loops_lab.js

```diff
@@ -0,0 +1,239 @@
+  if (num % 2 === 0){
+  } else{
+      console.log (num);
+  }
+}
+//
+// // // 5. Without running/executing your code, how many times will the loop below run? Explain why.
+// // //
+// // // ```js
+// // let i = 5;
+// //
+// // while (i > 3) {
+// //     i += 1
+// // }
+// // // //
+// // // // // Your explanation here
+``` the code would not print anything because there is no console.log in the curly brackets.
+// ```
+// //
+// // // 6. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+// //
+for(let x = 1; x <= 100; x++){
+  if(x % 3 === 0){
+    console.log('Fizz');
+  } else if ( x % 5 === 0){
+    console.log('Buzz');
+  } else {
+    console.log(x);
+  }
+}
+
+// // // // 7. Modify your program to log "FizzBuzz", for numbers that are divisible by both 3 and 5 (still log "Fizz" or "Buzz" for numbers divisible by only one of those).
+// //
+for(let x = 1; x <= 100; x++){
+  if(x % 3 === 0 && x % 5 === 0){
+    console.log('FizzBuzz');
+  } else if ( x % 5 === 0){
+    console.log('Buzz');
+  } else if ( x % 3 === 0 ){
+    console.log('Fizz')
+  }else {
+    console.log(x);
+  }
+}
+//
+// // // ### Bonus:
+// // //
+// // // 1. Write a program that would log the [lyrics of the song 99 Bottles of Beer](http://www.99-bottles-of-beer.net/lyrics.html). This is the first verse of the song:
+// // //
+// // //   ```
+// // //   99 bottles of beer on the wall, 99 bottles of beer.
+// // //   Take one down, pass it around, 98 bottles of beer on the wall.
+// // //   ```
+// // //
+// // //   This verse is repeated, each time with one less bottle, until the number of bottles is 0. When the number of bottles is 2, the verse is:
+// // //
+// // //   ```
+// // //   2 bottles of beer on the wall, 2 bottles of beer.
+// // //   Take one down, pass it around, 1 bottle of beer on the wall.
+// // //   ```
+// // //
+// // //   In the last line, the word bottles (plural), is  replaced with bottle (singular)
+// // //
+// // //   When the number of bottles is 1, the verse is:
+// // //
+// // //   ```
+// // //   1 bottle of beer on the wall, 1 bottle of beer.
+// // //   Take one down, pass it around, No more bottle of beer on the wall.
+// // // //   ```
+let init = 4;
+let fin = 1;
+let dinit = init - fin;
+while(init > 0){
+    console.log(init + ' bottle of beer on the wall,' + init + 'bottle of beer. Take one down, pass it around,' + dinit + ' bottles of beer on the wall.')
+    init = init - 1
+    if (init > 1){
+      console.log(init + ' bottle of beer on the wall,' + init + 'bottle of beer. Take one down, pass it around,' + dinit + ' bottles of beer on the wall.')
+      init = init -1
+    }
+  }
+  init --
+// // I am going to continue working on this...
+
+for(let init = 99, consis = 1; init > 0; init--){
```

Comment:
Good job with experimenting with two variables inside for loop. It's interesting usage although declaring consis outside of for loop is fine since we are not really looping using consis variable.